WebApplication/Backend/Util/MyDbContext.cs

```diff
@@ -20,6 +20,8 @@ public class MyDbContext : DbContext
         public DbSet<MedicineType> MedicineTypes { get; set; }
         public DbSet<Medicine> Medicine { get; set; }
 
+        public DbSet<MedicineGEA> MedicineGEA { get; set; }
```

Comment:
This is what i was saying in comment above. No need for MedicineGEA when Medicine is already created for need of gea team and it works fine.